cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -420,8 +420,8 @@ instance ToJSONKey MA.AssetName where
 instance ToJSON MA.ValidityInterval where
   toJSON vi =
     Aeson.object $
-        [ "validFrom" .= x | x <- mbfield (MA.validFrom vi) ]
-     ++ [ "validTo"   .= x | x <- mbfield (MA.validTo   vi) ]
+        [ "invalidBefore"    .= x | x <- mbfield (MA.invalidBefore    vi) ]
+     ++ [ "invalidHereafter" .= x | x <- mbfield (MA.invalidHereafter vi) ]
```

Comment:
I'm changing these fields, is that breaking something?

Reply:
Not for the benchmarking team.